news/2024/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: post
+title: "The Followpoint: xootynator, the CEO of Full Area"
+date: 2024-09-16 16:00:00 +0000
+series: The Followpoint
+---
+
+Get ready for an artistic spin with xootynator, the sorceress of full area!
+
+![](/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/banner.jpg)
+
+Guess what? The Followpoint is back to bring you another banger, an interview with [xootynator](https://osu.ppy.sh/users/3717598)! You likely heard of her at least once because of her community contributions (such as tournament staffing or content making), tournament participation, arts or her unique playstyle. And today we have a chance to learn more about her and her osu! journey.
```

Comment:
i might be completely in the wrong here but are you sure that she's known for tournament staffing? afaik she's a really active player but not staffer

Reply:
also, i dont think there's a need for a bracket here if you are already listing everything
```suggestion
Guess what? The Followpoint is back to bring you another banger, an interview with [xootynator](https://osu.ppy.sh/users/3717598)! You likely heard of her at least once because of her community contributions such as tournament staffing, content making, tournament participation, arts or her unique playstyle. Today we have a chance to learn more about her and her osu! journey.
```